Makefile

```diff
@@ -95,6 +94,10 @@ docker-manifest: docker-push ## Push docker multi-arch manifest.
 	docker manifest annotate ${IMG} ${IMG}-arm64 --arch=arm64
 	docker manifest push ${IMG}
 
+.PHONY: lint
+lint: golangci-lint ## Lint the codebase
+	$(GOLANGCI_LINT) run -v --go=1.19 --timeout 3m0s
```

Comment:
Can we use the golang version as a variable?
Something like GOLANG_VERSION=1.19?